spec/rubocop/cop/ezcater/feature_flag_active_spec.rb

```diff
@@ -0,0 +1,39 @@
+# encoding utf-8
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Ezcater::FeatureFlagActive, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:msgs) { [described_class::MSG] }
+
+  it "accepts a call with a tracking_id" do
+    source = %[EzFF.active?("FlagName", tracking_id: "user:23456")]
+    inspect_source(source)
+    expect(cop.offenses).to be_empty
+  end
+
+  it "accepts a call with an identifiers array" do
+    source = %[EzFF.active?("FlagName", identifiers: ["user:23456", "user:12345"])]
+    inspect_source(source)
+    expect(cop.offenses).to be_empty
+  end
+
+  it "errors when neither expected key is present" do
+    source = %[EzFF.active?("FlagName", bad_key: "user:123", other_bad_key: ["user:123", "user:456"])]
+    inspect_source(source)
+    expect(cop.messages).to match_array(msgs)
+  end
+
+  it "errors when only one argument is passed" do
+    source = %[EzFF.active?("FlagName")]
+    inspect_source(source)
+    expect(cop.messages).to match_array(msgs)
+  end
+
+  it "works with the full module name" do
+    source = %[EzcaterFeatureFlag.active?("FlagName")]
```

Comment:
Why does the full module behave differently?

Reply:
It doesn't, just wanted to ensure we're catching this behavior whether someone uses the full module name or the alias when they call the method.

Reply:
Ugh, misread.  NM.